tests/cucumber/e2e/stepDefinations/addmovieContext.js

```diff
@@ -0,0 +1,39 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+// import expect for assertion
+const { expect } = require("@playwright/test");
+const addBtn = `//button[@id="titleaddBtn"]`;
+const form = `//div[@id="myDropdown"]`;
```

Comment:
```suggestion
const form = `//div[@id="myDropdown"]`;
```
what do you mean by form variable? using CSS selector ID would be better in such cases.